app/Providers/EventServiceProvider.php

```diff
@@ -50,5 +53,15 @@ public function register()
         $this->app->bind(AddMemberActionForFinishedEvent::class, function (Application $app) {
             return new AddMemberActionForFinishedEvent($app->tagged('eventaction.applicator'));
         });
+
+        $this->app->bind(WebsiteUpdater::class, function (Application $app) {
+            return new WebsiteUpdaterThroughGithubActions(
+                new Client([
```

Comment:
Misschien persoonlijke voorkeur dit maar kunnen we deze `Client` class aliases naar `HttpClient` ofzo, dat is wat specifieker